scripts/windows/setup_project.bat

```diff
@@ -234,15 +222,11 @@ CALL :install_websockets %config_type% %generator%
 
 ::build the project with the selected options
 pushd %build_dire%
-if %use_depth_compute_stubs%==1 (
-   cmake -G %generator% -DWITH_PYTHON=on -DCMAKE_PREFIX_PATH="%deps_install_dir%\glog;%deps_install_dir%\protobuf;%deps_install_dir%\libwebsockets" %source_dir% -DUSE_DEPTH_COMPUTE_STUBS=ON -DCMAKE_BUILD_TYPE=%config_type%
-   ) else (
-		if %use_depth_compute_opensource%==1 (
-			cmake -G %generator% -DWITH_PYTHON=on -DCMAKE_PREFIX_PATH="%deps_install_dir%\glog;%deps_install_dir%\protobuf;%deps_install_dir%\libwebsockets" %source_dir% -DUSE_DEPTH_COMPUTE_OPENSOURCE=ON -DCMAKE_BUILD_TYPE=%config_type%
-		) else (
-			cmake -G %generator% -DWITH_PYTHON=on -DCMAKE_PREFIX_PATH="%deps_install_dir%\glog;%deps_install_dir%\protobuf;%deps_install_dir%\libwebsockets" %source_dir% -DCMAKE_BUILD_TYPE=%config_type%
-		) 
-)
+if %use_depth_compute_opensource%==1 (
+    cmake -G %generator% -DWITH_PYTHON=on -DCMAKE_PREFIX_PATH="%deps_install_dir%\glog;%deps_install_dir%\protobuf;%deps_install_dir%\libwebsockets" %source_dir% -DUSE_DEPTH_COMPUTE_OPENSOURCE=ON -DCMAKE_BUILD_TYPE=%config_type%
```

Comment:
On host (Windows, Linux, etc.) the  depth compute is no longer needed. Please remove flag DUSE_DEPTH_COMPUTE_OPENSOURCE=ON.

Reply:
Also remove the option from setup_project script.

Reply:
Solved